linera-execution/src/pricing.rs

```diff
@@ -9,19 +9,42 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 /// A collection of costs associated with blocks in validators.
-#[derive(Eq, PartialEq, Hash, Clone, Debug, Default, Serialize, Deserialize, InputObject)]
+#[derive(Eq, PartialEq, Hash, Clone, Debug, Serialize, Deserialize, InputObject)]
 pub struct Pricing {
```

Comment:
Perhaps we could discuss a new name such as `Resource(Control)Policy` if this is going to include limits. @afck ?

Reply:
Ok, I renamed it `ResourceControlPolicy`.
I also renamed `pricing` as `policy` when appropriate.
I kept the `PricingError` since it occurs only for monetary reasons.